apps/e2e-test/features/publisher-camera-stream.feature

```diff
@@ -0,0 +1,27 @@
+@publisher
+Feature: Publisher Camera Streaming
+    As a publisher
+    I want to do live streaming for an event with Camera
+
+    Scenario: Publisher should be presented with different streaming control buttons when stream is live
+        Given a publisher is on the "preview" page
+        When the publisher clicks on the "go live button"
+        Then the publisher should be navigated to "stream" page
+        
+        And the "stop button" should be displayed
+        And the "stop button" should be enabled
+        And the "stop button" text should be "STOP"
+
+        And the "add source button" should be displayed
+        And the "add source button" should be enabled
+        And the "add source button" text should be "Add Source"
+
+        And the "invite button" should be displayed
+        And the "invite button" should be enabled
+        And the "invite button" text should be "Invite viewers"
+
+        And the "streaming header" should be displayed with default values
```

Comment:
We have 3 variations for this step:

- And the "streaming header" should be displayed with default values
  **Notes: This will verify the header components state and text with the default values

- And the "streaming header" should be displayed with only values
       | stream info button |   enabled         |
       | microphone status  |   Off                |
  **Notes: This will verify the header components state and text with the below values only and default values are not verified

- And the "streaming header" should be displayed with following values
      | camera status      |   Off                |
  **Notes: This will verify the header components state and text with the below values  and default values

Reply:
So will you put these instructions in README.md or Contribution.md?